template/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Netlify Release
```

Comment:
Should we change this to a name that starts with a verb?
```suggestion
name: Deploy to Netlify
```

Reply:
And should we add a line break the below line as well?

Reply:
50c2b66 ✅